src/expr/macro/src/gen.rs

```diff
@@ -781,20 +782,25 @@ impl FunctionAttr {
                 use risingwave_common::buffer::Bitmap;
                 use risingwave_common::estimate_size::EstimateSize;
 
+                use risingwave_expr::expr::Context;
                 use risingwave_expr::Result;
                 use risingwave_expr::aggregate::AggregateState;
                 use risingwave_expr::codegen::async_trait;
 
-                #[derive(Clone)]
+                let context = Context {
+                    return_type: agg.return_type.clone(),
+                    arg_types: agg.args.arg_types().to_owned(),
+                };
```

Comment:
Will it introduce additional overhead for other agg function impls? Though it seems not huge.

Reply:
It may introduce some minor one-off overhead regarding other agg func, which happens only when the `Agg` is created. For the `return_type` interface, one more memory access is required, but the interface should not be called frequently. So I think the overhead is negligible.